api/API_VERSIONING.md

```diff
@@ -67,6 +67,8 @@ experience a backward compatible break on a change. Specifically:
     may be granted for scenarios in which these stricter conditions model behavior already implied
     structurally or by documentation.
 
+* Messages marked as [#alpha:] are excluded from the backwards compatibility guarantees.
```

Comment:
Please put this in the bulletted list below, which lists the other items that have an exception.